cli/cmd/options.go

```diff
@@ -469,12 +469,6 @@ func makeInjectFlags(defaults *l5dcharts.Values) ([]flag.Flag, *pflag.FlagSet) {
 				return nil
 			}),
 
-		flag.NewBoolFlag(injectFlags, "disable-identity", defaults.Proxy.DisableIdentity,
-			"Disables resources from participating in TLS identity", func(values *l5dcharts.Values, value bool) error {
-				values.Proxy.DisableIdentity = value
-				return nil
-			}),
```

Comment:
Should we retain this flag and print an error?

Reply:
That would makes sense to me if we'd be removing some functionality, but this has been broken for a long time, so why not just get rid of it?

Reply:
This changed in 2.11, so it wasn't that long ago.. we just missed this as a part of the 2.11 release. I don't feel strongly, though.

Reply:
Ok, I'll add it back, erroring with a message.